test/data/fake_samples.tsv

```diff
@@ -2,4 +2,4 @@ validator types:	number	noop	onto	enum	onto	user field	number
 test name field	Depth bgs	Country	biome	material	feature	user field	elevation start	elevation unit	size
 s1	0	Germany	ENVO:01000221	Gas	ENVO:01000002		123	cm	1
 s2	1.04	Uzbekistan		Soil		blip	3	feet	1
-s3	299	Cambodia	temperate woodland biome		ENVO:00002226	blap blork	5.9	meters	1
```

Comment:
I think we should leave this as "temperate woodland biome", its to test the functionality of looking up the term id given the term name from the OntologyAPI, which I think we added support for on certain terms

Reply:
👍